module/behaviour/strategy/SoccerStrategy/src/SoccerStrategy.hpp

```diff
@@ -41,15 +41,12 @@ namespace module::behaviour::strategy {
             Config() = default;
             NUClear::clock::duration ball_last_seen_max_time{};
             NUClear::clock::duration goal_last_seen_max_time{};
-
             float ball_search_walk_start_speed = 0.0f;
             float ball_search_walk_stop_speed  = 0.0f;
             float ball_search_walk_slow_time   = 0.0f;
-
             Eigen::Vector2d start_position_offensive{Eigen::Vector2d::Zero()};
             Eigen::Vector2d start_position_defensive{Eigen::Vector2d::Zero()};
-            bool is_goalie = false;
-
+            bool is_goalie                         = false;
```

Comment:
For 2022 we'll only have Goalie and Field Player roles right? In future we may want to change this to an enum for different playstyles, although that'll likely end up being handled by director.

No issue with this implementation as a bool at the moment imo just thought I'd ask to confirm those are the only two roles we intend on using this year

Reply:
Yeah I can't see us having the capability to have more than goalie and field player. We don't have tested goalie behaviour atm afaik. Director is more important once we have something working as a base for playing games.